python/Ganga/Core/GangaRepository/GangaRepository.py

```diff
@@ -205,7 +205,7 @@ def _make_empty_object_(self, this_id, category, classname):
         #obj.__init__()
         obj.setNodeData({})
         obj.setNodeAttribute('id', this_id)
-
+        obj._setFlushed()
```

Comment:
looking up the object type used to be much more difficult.
(frankly you can always use a lookup in the GPI for the proxied class and strip that to create an instance so all the dancing around in plugins I dislike)

Also the object used to be something distinctly different but caused more difficulty it used to be a new object which wasn't fully initialized (and this caused many, many errors).
As it stands the Object is made _empty_ as there is no dictionary associated with schema attributes which will trigger a load in the ```__get__``` method. Returning a vanilla object here will just make the object look like a new GangaObject and never trigger a load from disk until disk access is requested.

I'm much more keen to move to a framework where each object explicitly knows that it's been loaded or not based upon an internal boolean. This avoids all of this conditional object inspection and allows us to simply the question of if the thing ever loaded from the disk or not.